index.html

```diff
@@ -0,0 +1,167 @@
+                <p class="venue">@National Meseum of Korea. <em>Stream live @www.global-streaming.org</em</p>
+              </div>
+            </div>
+        </section>
+        <!-- events start-->
+         <section class="cards" id="event-categories"> 
+         <h4>Upcoming Events</h4>
+         <hr>
+         <div class="card-content event-card">
+         <div class="card">
+            <div class="title-img">
+                <div class="image">
+                    <span class="fa-solid fa-newspaper"></span>
+                </div>
+                <p class="title1">News</p>
+            </div>
+            <p class="card-description ">
+            <a href="#category-display" id="news" class="clicable-card news">Information that is published in newspapers 
+                and broadcast on radio and television about 
+                recent events in the country or world or in a particular area of activity.</a>
+            </p>
+        </div>
+         <div class="card">
+            <div class="title-img">
+                <div class="image">
+                    <span class="fa fa-music me"></span>
+                </div>
+                <p class="title1">Entertainment</p>
+            </div>
+            <p class="card-description ">  
+            <a href="#category-display" id="entertainment" class="clicable-card entertainment">These events are form of 
+                activities that holds the attention and interest of an 
+                audience or gives pleasure and delight.</a>
+            </p>
+         </div>
+         <div class="card">
+            <div class="title-img">
+                <div class="image">
+                    <span class="fa fa-book me"></span>
+                </div>
+                <p class="title1"> Education</p>
+            </div>
+            <p class="card-description "> 
+            <a href="#category-display" id="education" class="clicable-card education"> Events purposeful activity directed at achieving 
+                certain aims, such as transmitting 
+                knowledge or fostering skills and character traits</a>
+            </p>
+         </div>
+         <div class="card">
+            <div class="title-img">
+                <div class="image">
+                    <span class="fa-sharp fa-solid fa-basketball"></span>
+                </div>
+                <p class="title1"> Sports</p>
+            </div>
+            <p class="card-description ">  
+            <a href="#category-display" id="sports" class="clicable-card sports"> Events involving physical exertion and skill 
+                 in which an individual or team competes 
+                 against another or others for entertainment</a></p>
+         </div>
+        </div>
+        <a href="#category-display" class="see-category"> See Selected events category</a> 
+        </section> 
+      <!-- category-display card section---------------- -->
+      <section id="category-display"></section>  
```

Comment:
- [ ] This section should be similar to the one in the design, this is how it looks on your website:

  ![image](https://user-images.githubusercontent.com/20567503/198014514-f8b30b83-82d6-4b85-a554-6e8604c8bb06.png)

  This is how it should look:

  ![image](https://user-images.githubusercontent.com/20567503/198014553-7a0313f0-0556-4ac6-a546-ad8b115253e4.png)
  
  Kindly try to follow the design provided as close as possible, this means that you need to change the title style, and each card styling as well.